src/routes/diluent.test.js

```diff
@@ -0,0 +1,89 @@
+import express from 'express';
+import request from 'supertest';
+import passport from 'passport';
+import AnonymousStrategy from 'passport-anonymous';
+import bodyParser from 'body-parser';
+
+import diluent from './diluent';
+import database from '../modules/database';
+
+describe('diluent route resource', () => {
+  const app = express();
+
+  passport.use(new AnonymousStrategy());
+  app.use(bodyParser.json());
+  app.use(bodyParser.urlencoded({ extended: false }));
+  app.use(diluent);
+
+  afterAll(() => {
+    database.sequelize.close();
+  });
+
+  const date = new Date();
+
+  const mockData = {
+    name: 'Grosser Stuff' + date,
+    slug: 'gs',
+    code: 'GS',
+    density: '1.2610'
```

Comment:
This should be `1.2610` as a number, no?

Reply:
eslint doesn't like trailing zeros in numbers

Reply:
Oh, does `1.261` work though?